helmexec/runner.go

```diff
@@ -15,10 +15,15 @@ type Runner interface {
 }
 
 // ShellRunner implemention for shell commands
-type ShellRunner struct{}
+type ShellRunner struct {
+	Dir string
+}
 
 // Execute a shell command
 func (shell ShellRunner) Execute(cmd string, args []string) ([]byte, error) {
 	preparedCmd := exec.Command(cmd, args...)
+	if shell.Dir != "" {
+		preparedCmd.Dir = shell.Dir
```

Comment:
What do you think about setting `preparedCmd.Dir` unconditionally? It's default value is the empty string.

Reply:
I believe we shouldn't, because we use it to run helm commands and it assumes Dir unspecified today.

Reply:
What I meant is that the condition is not necessary:

if shell.Dir is empty, assign (it won't change the default of preparedCmd which is empty)
if shell.Dir is not empty, assign (as you you in the condition)

Reply:
Ah make sense! I'll fix it asap. Thanks for your review!